TROUBLESHOOTING.md

```diff
@@ -18,7 +18,7 @@ version we tried to install.  E.g. on conda you might need to do:
 
 ```
 pip uninstall numpy
-pip install numpy==1.14.5 # or whatever version was attempted to be installed by the wheel
+pip install numpy==1.15.1 # or whatever version was attempted to be installed by the wheel
```

Comment:
Same here, needed?

Reply:
I think so

Reply:
Yes, ok, DAI has 1.15.1, so is good change.